SimG4CMS/Forward/src/CastorTestAnalysis.cc

```diff
@@ -41,20 +41,19 @@ CastorTestAnalysis::CastorTestAnalysis(const edm::ParameterSet &p) {
   stepNtFileName           = m_Anal.getParameter<std::string>("StepNtupleFileName");
   eventNtFileName          = m_Anal.getParameter<std::string>("EventNtupleFileName");
 
-  if (verbosity > 0)
-   std::cout<<std::endl;
-   std::cout<<"============================================================================"<<std::endl;
-   std::cout << "CastorTestAnalysis:: Initialized as observer"<< std::endl;
-   if (doNTcastorstep  > 0){
-     std::cout <<" Step Ntuple will be created"<< std::endl;
-     std::cout <<" Step Ntuple file: "<<stepNtFileName<<std::endl;
-   }
-   if (doNTcastorevent > 0){
-     std::cout <<" Event Ntuple will be created"<< std::endl;
-     std::cout <<" Step Ntuple file: "<<stepNtFileName<<std::endl;
-   }
-   std::cout<<"============================================================================"<<std::endl;
-   std::cout<<std::endl;
```

Comment:
Actually it looks like everything up to here is meant to be in the if statement from line 44.